variables.tf

```diff
@@ -156,6 +156,20 @@ variable "max_unavailable" {
   default     = 0
 }
 
+variable "maintenance_window" {
+  description = "The time windows when GKE can be allowed to perform maintenance ops like version upgrade, repair, scheduled maintenance etc. GKE requires the total sum of allowed hours to be at least 48 hours per 32 days - with no single window duration being shorter than 4 hours.\n\nThe \"_time_utc\" values are allowed in 24-hour \"HH:MM\" format only and must be in UTC timezone. The \"days_of_week\" value is a CSV string containing 2-letter notations of days in a week in capital letters. Check the default values for reference. Note that, the timezone conversions from UTC do not impact the day values.\n\nKNOWN ISSUE: If the \"start_time_utc\" is as late in the day as pushing \"end_time_utc\" over to the next day in UTC, then \"end_time_utc\" will become smaller than \"start_time_utc\" - this is disallowed by Terraform. Therefore, both \"start_time_utc\" and \"end_time_utc\" must be within the same single day in UTC."
+  type = object({
+    start_time_utc = string
+    end_time_utc   = string
+    days_of_week   = string
+  })
+  default = {
+    start_time_utc = "19:00"          # implies 3AM in MYT
+    end_time_utc   = "23:00"          # implies 7AM in MYT
+    days_of_week   = "MO,TU,WE,TH,FR" # implies MO,TU,WE,TH,FR in MYT - remains unchanged by timezone conversion
```

Comment:
We should only give weekends in the days of week, not working days.

Reply:
My thinking was, it should be during early mornings of working days so that engineers are available in office next morning in case something went wrong.

Reply:
@syedrakib it should be Sunday to Thursday. Fri 7pm UTC will be Sat 3am MYT

Reply:
See the comment "remains unchanged by timezone conversion"

This is the result of the above default values

![image](https://user-images.githubusercontent.com/975768/118589833-c53ca300-b7d3-11eb-9925-f813ff779eeb.png)